### PR TITLE
fix : modify the version of ks core components

### LINKED ISSUE #4437: ks-console, ks-apiserver and ks-controller-manager version error
**Versions Used**
KubeSphere:  `v3.2.0`

**How To Reproduce**
image
image
image

/kind bug
/priority low
/milestone v3.2

### PR DESCRIPTION
**Pull request by 123liubao**
Signed-off-by: 123liubao <chenly_61@163.com>

### What type of PR is this?

/kind bug

### Which issue(s) this PR fixes:

Fixes #4437

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note
None
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -127,7 +127,7 @@ container-cross-push: ; $(info $(M)...Begin to build and push.)  @ ## Build and
 
 helm-package: ; $(info $(M)...Begin to helm-package.)  @ ## Helm-package.
 	ls config/crds/ | xargs -i cp -r config/crds/{} config/ks-core/crds/
-	helm package config/ks-core --app-version=v3.1.0 --version=0.1.0 -d ./bin
+	helm package config/ks-core --app-version=v3.2.0 --version=0.1.0 -d ./bin
```
Comment (RolandMa1986) [inline Makefile:130]: Maybe we can change the version to a  `make ` parameter. The default value is set to v3.2.0.

Comment (123liubao) [reply Makefile]: Thank you for your suggestion, I have updated the code.

**Comment by zryfish** [comment]
@123liubao  Thanks.
/lgtm
/approve

**Pull request merged**